templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -4,6 +4,20 @@
 
 set -mveuo pipefail
 
+if [ "${GITHUB_REF##refs/heads/}" = "${GITHUB_REF}" ]
```

Comment:
I'm curious about what this check is doing. I guess I would have imagined we'd check something like `"$GITHUB_REF" = refs/heads*`?

Reply:
Well the `##` removes `refs/heads/` from the front if it is there, and in that case the strings stop to be equal. So yes.

Reply:
Hmm it seems like such an indirect way of checking that, no?

Reply:
It is compatible with the all bourn shell flavors, afaik.